manifests/windows/alias.pp

```diff
@@ -0,0 +1,28 @@
+class classroom::windows::alias {
+
+  assert_private('This class should not be called directly')
+
+  $ps_module_path = 'C:/Users/Administrator/Documents/WindowsPowerShell/Modules'
+  $alias_dir      = "${ps_module_path}/alias"
+  $psm_file       = 'alias.psm1'
+  $psd_file       = 'alias.psd1'
```

Comment:
@Iristyle That's because the .psd1 file is created by the exec resource? /cc @sshastri